### PR TITLE
New Vending Machine - SCC Encryption Key Vendor

### PR DESCRIPTION
**Pull request by courierbravo**
Adds a new vending machine, along with a modified sprite and a refill canister for it.
Removes the boba from the dinnerware vendor due to it not making sense.

### CODE REVIEW

**Comment by courierbravo** [comment]
!review

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		/obj/item/device/vending_refill/cutlery = 1,
 		/obj/item/device/vending_refill/robo = 1,
 		/obj/item/device/vending_refill/battlemonsters = 1,
+		/obj/item/device/vending_refill/encryption
```
Comment (DreamySkrell) [inline code/game/machinery/vending_types.dm:43]: = 1,

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -1808,3 +1808,33 @@
...
+		/obj/item/device/encryptionkey/heads/rd = 1,
+		/obj/item/device/encryptionkey/heads/xo = 1,
+		/obj/item/device/encryptionkey/headset_operations_manager = 1,
+		/obj/item/device/encryptionkey/headset_com = 5,
+		/obj/item/device/encryptionkey/headset_cargo = 5,
+		/obj/item/device/encryptionkey/headset_eng = 5,
+		/obj/item/device/encryptionkey/headset_med = 5,
+		/obj/item/device/encryptionkey/headset_sci = 5,
+		/obj/item/device/encryptionkey/headset_sec = 5,
+		/obj/item/device/encryptionkey/headset_service = 5,
```
Comment (DreamySkrell) [inline code/game/machinery/vending_types.dm:1834]: I think would be nice to have more basic department keys, like 10 maybe

Comment (courierbravo) [reply code/game/machinery/vending_types.dm:1834]: 5 max is plenty to have on tap. I have, however, increased the canister refill capacity by x3, which allows for nearly 1.5 full refills from empty.

**Review by DreamySkrell — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**